src/main/java/uk/co/downthewire/jLTE/fading/GenerateFading.java

```diff
@@ -40,13 +40,14 @@
 	public static void main(String[] args) throws IOException, ConfigurationException {
 		Configuration configuration = new PropertiesConfiguration("system.properties").interpolatedConfiguration();
 
-		String fadingDirectory = configuration.getString(FieldNames.FADING_PATH);
+		//String fadingDirectory = configuration.getString(FieldNames.FADING_PATH);
+        String fadingDirectory = BuildPath.getFadingPath(configuration);
 
-		int numUEs = 1150;
+		int numUEs = configuration.getInt(FieldNames.NUM_UES);
```

Comment:
These are good changes. I never got round to making this class as user-friendly as it should be. Thanks